src/VisualStudio/Core/Def/ColorSchemes/Enhanced.xml

```diff
@@ -0,0 +1,810 @@
+<Themes>
+    <!-- IMPORTANT: Any changes to this file need to be synced with the ColorSchemeApplier.ForegroundColorDefaulter. -->
+    <Theme Name="Light" GUID="{de3dbbcd-f642-433c-8353-8f1df4370aba}">
+        <Category Name="Roslyn Text Editor MEF Items" GUID="{75a05685-00a8-4ded-bae5-e7a50bfa929a}">
+            <Color Name="preprocessor text">
+                <Foreground Type="CT_RAW" Source="FF000000" />
+            </Color>
+            <Color Name="punctuation">
+                <Foreground Type="CT_RAW" Source="FF000000" />
+            </Color>
+            <Color Name="string - verbatim">
+                <Foreground Type="CT_RAW" Source="FF800000" />
+            </Color>
+            <Color Name="keyword - control">
+                <Foreground Type="CT_RAW" Source="FF8F08C4" />
+            </Color>
+            <Color Name="operator - overloaded">
+                <Foreground Type="CT_RAW" Source="FF74531F" />
+            </Color>
+            <Color Name="class name">
+                <Foreground Type="CT_RAW" Source="FF2B91AF" />
+            </Color>
+            <Color Name="delegate name">
+                <Foreground Type="CT_RAW" Source="FF2B91AF" />
+            </Color>
+            <Color Name="enum name">
+                <Foreground Type="CT_RAW" Source="FF2B91AF" />
+            </Color>
+            <Color Name="interface name">
+                <Foreground Type="CT_RAW" Source="FF2B91AF" />
+            </Color>
+            <Color Name="module name">
+                <Foreground Type="CT_RAW" Source="FF2B91AF" />
+            </Color>
+            <Color Name="struct name">
+                <Foreground Type="CT_RAW" Source="FF2B91AF" />
+            </Color>
+            <Color Name="type parameter name">
+                <Foreground Type="CT_RAW" Source="FF2B91AF" />
+            </Color>
+<!--
```

Comment:
The XML wouldn't validate with the ColorCompilers .xsd. I wanted the list of classifications to be exhaustive for each theme, so I commented them out. Since I am switching to reading these .xml files ourselves, I can loosen this restriction and allow them to be empty.